.github/styles/Vocab/Docs/accept.txt

```diff
@@ -122,4 +126,7 @@ unmount(|s|ing|ed)
 (Velero|velero)
 XGBoost
 yakcl
-(zookeeper|Zookeeper)
+Zookeeper
+zookeeper
+ZooKeeper
```

Comment:
are you _sure_ you want to add ZooKeeper? shouldn't it remain consistent?

Reply:
resolved in #4001